docs/tutorial/exporting_blt_targets.rst

```diff
@@ -0,0 +1,74 @@
+.. # Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. # other BLT Project Developers. See the top-level COPYRIGHT file for details
+.. # 
+.. # SPDX-License-Identifier: (BSD-3-Clause)
+
+Exporting BLT-created Targets
+=============================
+
+BLT provides several built-in targets for commonly used libraries:
+
+ * ``mpi`` (when ``ENABLE_MPI`` is on)
+ * ``openmp`` (when ``ENABLE_OPENMP`` is on)
+ * ``cuda`` and ``cuda_runtime`` (when ``ENABLE_CUDA`` is on)
+ * ``hip`` and ``hip_runtime`` (when ``ENABLE_HIP`` is on)
+
+These targets can be made exportable in order to make them available to users of
+your project via CMake's ``install()`` command.  Setting BLT's ``BLT_EXPORT_THIRDPARTY``
+option to ``ON`` will mark all active targets in the above list as ``EXPORTABLE``
+(see the ``blt_import_library()`` documentation for more info).
+
+.. note:: If a target in your project is added to an export set, any of its dependencies
+  marked ``EXPORTABLE`` must be added to the same export set.  Failure to add them will
+  result in a CMake error in the exporting project.
+
+Typical usage of the ``BLT_EXPORT_THIRDPARTY`` option is as follows:
+
+.. code-block:: cmake
+
+    # BLT configuration - enable MPI
+    set(ENABLE_MPI ON CACHE BOOL "")
+    # and mark the subsequently created MPI target as exportable
+    set(BLT_EXPORT_THIRDPARTY ON CACHE BOOL "")
```

Comment:
Add a comment that (and/or show here) that these need to be set before SetupBLT.cmake is called